libproxy/cmake/modules/pacrunner_duktape.cmk

```diff
@@ -1,11 +1,19 @@
+option(WITH_DUKTAPE "Search for Duktape package" ON)
+
 if(WIN32)
-  set(DUKTAPE_FOUND 0)
+  set(DUKTAPE_FOUND 0)  
+  if(WITH_DUKTAPE)
+    find_package(duktape QUIET)
```

Comment:
If you are using the `vcpkg` definition, it looks like it just [uses pkg-config internally](https://github.com/microsoft/vcpkg/blob/5251291e1623eabf5c8f275f456bc95f983cd8eb/ports/duktape/duktapeConfig.cmake.in#L11). So I wonder If we could re-use the config the non-`WIN32` definition.